libkbfs/disk_block_cache_test.go

```diff
@@ -608,7 +604,7 @@ func TestSyncBlockCacheStaticLimit(t *testing.T) {
 	t.Log("Test that disk cache eviction works when we hit the static limit.")
 	cache, config := initDiskBlockCacheTest(t)
 	standardCache := cache.syncCache
-	defer shutdownDiskBlockCacheTest(standardCache)
```

Comment:
Good question, I don't know either.  Probably just someone typed it that way without thinking, and it just worked since the local cache implemented the generic interface.